docs/4.2/ssh_azuread.md

```diff
@@ -0,0 +1,190 @@
+# SSH Authentication with Azure Active Directory (AD)
+
+This guide will cover how to configure [Microsoft Azure Active Directory(https://azure.microsoft.com/en-us/services/active-directory/) to issue
+SSH credentials to specific groups of users with a SAML Authentication Connector. When used in combination with role
+based access control (RBAC) it allows SSH administrators to define policies
+like:
+
+* Only members of "DBA" Google group can SSH into machines running PostgreSQL.
+* Developers must never SSH into production servers.
+* ... and many others.
+
+The following steps configure an example SAML authentication connector matching AzureAD groups with security roles.  You can choose to configure other options.
+
+!!! warning "Version Warning"
+
+    This guide requires an enterprise version of Teleport. The open source
+    edition of Teleport only supports [Github](admin-guide.md#github-oauth-20) as
+    an SSO provider.
+
+## Prerequisites:
+
+Before you get started you’ll need:
+
+- An Enterprise version of Teleport v4.2 or greater, downloaded from [https://dashboard.gravitational.com/](https://dashboard.gravitational.com/web/). 
+- Azure AD admin account with access to creating non-gallery applications (P2 License)
+- Register one or more users in the directory
+- Create at least two security groups in AzureAD and assign one or more users to each group
+
+
+
+## Configure Azure AD
+
+1. Select Enterprise Applications from the AzureAD Directory Home
+  ![Select Enterprise Applications From Manage](img/azuread/azuread-1-home.png)
+  
+2. Select New application
+  ![Select New Applications From Manage](img/azuread/azuread-2-newapp.png)
+  
+3. Select a Non-gallery application
+   ![Select Non-gallery application](img/azuread/azuread-3-selectnongalleryapp.png)
+   
+4. Enter the display name (Ex: teleport)
+   ![Enter application name](img/azuread/azuread-4-enterappname.png)
+   
+5.Select properties under Manage and turn off User assignment required
+   ![Turn off user assignment](img/azuread/azuread-5-turnoffuserassign.png)
+   
+6. Select Single Sign-on under Manage and choose SAML
+   ![Select SAML](img/azuread/azuread-6-selectsaml.png)
+   
+7. Select to edit  Basic SAML Configuration
+   ![Edit Basic SAML Configuration](img/azuread/azuread-7-editbasicsaml.png)
+   
+8. Put in the Entity ID and Reply URL the same proxy url https://proxy1:3080/v1/webapi/saml/acs
```

Comment:
@benarent Do we have a default URL we use for replacements here? like https://<proxy-address> or whatever

Reply:
using the same example teleport.example.com